app/src/main/java/org/fossasia/openevent/general/search/SearchLocationActivity.kt

```diff
@@ -1,29 +1,42 @@
 package org.fossasia.openevent.general.search
 
+import android.annotation.SuppressLint
 import android.content.Intent
-import android.os.Bundle
-import android.view.MenuItem
+import android.content.pm.PackageManager
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.SearchView
-import kotlinx.android.synthetic.main.activity_search_location.search
-import org.fossasia.openevent.general.MainActivity
+import android.os.Bundle
 import org.fossasia.openevent.general.R
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_search_location.*
```

Comment:
I think the build is failing due to spot *, line13.

Reply:
```
 What went wrong:
Execution failed for task ':app:spotlessKotlin'.
> Error on line: 13, column: 1
  Exceeded max line length (120)
```

Reply:
Tried this but if I don't do it then other components of the activity is not accessible. Like progress bar etc

Reply:
Import them separately. Take an idea from PR #769

Reply:
Importing them separately will further increase its length
Increasing max length or disabling the check for this import (like in typescript) may solve this issue @liveHarshit  pls correct me

Reply:
But it is not the correct way to do changes in typescript and push them in the PR. Although, Mentors are also not accepting that same. Please look in PR #769 , Imports are added separately instead of import all at once. I think That's why build script was changed to not accepting *.

Reply:
ktlint is anti-bikeshedding. What you guys are discussing is bikehedding. The solution is to disable wildcard imports in your IDE. The steps are given in ktlint repo. 

Adding more imports will **not** affect max line size, it will increase the vertical size of the file. Line size checks the horizontal size of the file. 

Also, ktlint doesn't offer you to disable checks. Or else, everyone would just disable the checks and move on

Reply:
OK, so just checked. ktlint does not apply max length check on import statements. I personally downloaded this branch and applied ktlint. Only those lines are reported which have content more than of 120 characters, and not imports. Wildcard imports are disallowed and need to be changed

So @kshitijsuri90 Please work on it